packages/uui-copy/lib/uui-copy.element.ts

```diff
@@ -0,0 +1,175 @@
+import { defineElement } from '@umbraco-ui/uui-base/lib/registration';
+import { css, html, LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+import { UUIButtonElement } from '@umbraco-ui/uui-button/lib';
+import { UUICopyEvent } from './UUICopyEvent';
+import { demandCustomElement } from '@umbraco-ui/uui-base/lib/utils';
+import { LabelMixin } from '@umbraco-ui/uui-base/lib/mixins';
+
+/**
+ * @summary A button to trigger text content to be copied to the clipboard
+ * @element uui-copy
+ * @dependency uui-button
+ * @dependency uui-icon
+ * @fires {UUICopyEvent} copying - Fires before the content is about to copied to the clipboard and can be used to transform or modify the data before its added to the clipboard
+ * @fires {UUICopyEvent} copied - Fires when the content is copied to the clipboard
+ * @slot - Use to replace the default content of 'Copy' and the copy icon
+ */
+@defineElement('uui-copy')
+export class UUICopyElement extends LabelMixin('', LitElement) {
```

Comment:
Good that you extend the LabelMixin now, @warrenbuckley. I mentioned it to @nielslyngsoe and he was asking why we don't just extend the **UUIButtonElement** itself as we only ever have a `<uui-button></uui-button>` in the rendered output. If you extend from **UUIButtonElement**, you can remove many of the properties from **uui-copy** as they would already be present there.

Reply:
My question to you both @iOvergaard & @nielslyngsoe is that as this is not a typical Form Control and inherting/extending from UUIButton which in turn extends with the mixin `UUIFormControlMixin` which in turn already has a `value` field.

So can you let me still if you want me to extend from UUIButton, as it seems extra bloat to me to participate as a native form control when it will never be used as part of a form submission.